src/CompanyService.Business/Commands/Department/Interfaces/IGetDepartmentCommand.cs

```diff
@@ -9,14 +11,14 @@ namespace LT.DigitalOffice.CompanyService.Business.Commands.Department.Interface
     /// Provides method for getting department model by id.
     /// </summary>
     [AutoInject]
-    public interface IGetDepartmentByIdCommand
+    public interface IGetDepartmentCommand
     {
         /// <summary>
         /// Returns the department model with the specified id.
         /// </summary>
         /// <param name="departmentId">Specified id of department.</param>
         /// <returns></returns>
         /// <exception cref="Kernel.Exceptions.NotFoundException">Thrown when departmen is not found.</exception>
-        DepartmentInfo Execute(Guid departmentId);
+        OperationResultResponse<DepartmentResponse> Execute(GetDepartmentFilter filter);
```

Comment:
if you change the method signature change the documentation too

Reply:
Or delete it at all)

Reply:
Removed docs